app/views/articles/index.html.haml

```diff
@@ -13,7 +13,7 @@
         %td= article.title
         %td= article.text
         %td= link_to 'Show', article_path(article)
-        -if can? :create, Article
+        -if can? :update, Article
```

Comment:
use article not Article. you wanna know if the user can update this instance of an article